@navikt/core/react/src/overlays/floating-menu/Menu.stories.tsx

```diff
@@ -0,0 +1,600 @@
+import React, { useState } from "react";
+import { Menu } from "./Menu";
+
+export default {
+  title: "Utilities/FloatingMenu",
+  excludeStories: ["TickIcon", "classes"],
+  decorators: [
+    (Story) => (
+      <div>
+        {storyStyles}
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export const Styled = () => (
+  <MenuWithAnchor>
+    <Menu.Item className="item" onSelect={() => window.alert("undo")}>
+      Undo
+    </Menu.Item>
+    <Menu.Item className="item" onSelect={() => window.alert("redo")}>
+      Redo
+    </Menu.Item>
+    <Menu.Separator className="separator" />
+    <Menu.Item className="item" disabled onSelect={() => window.alert("cut")}>
+      Cut
+    </Menu.Item>
+    <Menu.Item className="item" onSelect={() => window.alert("copy")}>
+      Copy
+    </Menu.Item>
+    <Menu.Item className="item" onSelect={() => window.alert("paste")}>
+      Paste
+    </Menu.Item>
+  </MenuWithAnchor>
+);
+
+export const Submenus = () => {
+  const [open1, setOpen1] = React.useState(false);
+  const [open2, setOpen2] = React.useState(false);
+  const [open3, setOpen3] = React.useState(false);
+  const [open4, setOpen4] = React.useState(false);
+
+  return (
+    <MenuWithAnchor>
+      <Menu.Item className="item" onSelect={() => window.alert("undo")}>
+        Undo
+      </Menu.Item>
+      <Submenu open={open1} onOpenChange={setOpen1}>
+        <Menu.Item className="item" disabled>
+          Disabled
+        </Menu.Item>
+        <Menu.Item className="item" onSelect={() => window.alert("one")}>
+          One
+        </Menu.Item>
+        <Submenu open={open2} onOpenChange={setOpen2}>
+          <Menu.Item className="item" onSelect={() => window.alert("one")}>
+            One
+          </Menu.Item>
+          <Menu.Item className="item" onSelect={() => window.alert("two")}>
+            Two
+          </Menu.Item>
+          <Menu.Item className="item" onSelect={() => window.alert("three")}>
+            Three
+          </Menu.Item>
+          <Menu.Item className="item" onSelect={() => window.alert("four")}>
+            Four
+          </Menu.Item>
+          <Menu.Item className="item" onSelect={() => window.alert("five")}>
+            Five
+          </Menu.Item>
+          <Menu.Item className="item" onSelect={() => window.alert("six")}>
+            Six
+          </Menu.Item>
+        </Submenu>
+        <Submenu heading="Sub Menu" open={open3} onOpenChange={setOpen3}>
```

Comment:
Not able to reproduce myself, do you have a screen-recording?

Reply:
I accidentally found another bug when recording 😅 The "original" bug can be seen after about 5 secs:
https://github.com/navikt/aksel/assets/83693529/bfe801ad-7efe-4719-9743-a9d817c08538
The first few seconds shows a bug that occurs if you first open two submenus, then focus another app, then hover a different submenu.

Reply:
To reproduce, you have to move the mouse down quite fast, then keep it still. As soon as you move the mouse again, the menu opens as it should. Considering how hard it is to reproduce, it _might_ not be a real issue.

Reply:
Categorising the issue as "not a real issue" for now since it seems to be related to the small "grace"-delay before opening submenus and not a blocker for now.

Reply:
As for the "focus out of app" issue: This seems to be a windows-related issue since macos "freezes" the browser state when focusing another app, then resumes it when re-joining the browser causing no issues 🤔